### PR TITLE
Update product upgrades - Umbraco Cloud

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update product upgrades:

- [x] Write shorter sentences
- [x] e.g. --> for example
- [x] i.e. --> that is
- [x] html --> HTML 

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco Cloud

### CODE REVIEW

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR! 🙌 

I have few suggestions so I was wondering if you can take a loot at them and let me know what you think? 😊

**Comment by erikjanwestendorp** [comment]
@alina-tincas Thanks for your review, just updated the PR with your suggestions 😄

File: umbraco-cloud/product-upgrades/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ When minor upgrades are available, you will need a Development environment on yo
 * Umbraco Cloud reserves the right to roll out an emergency product fix to fix a critical issue at any time
 
 {% hint style="info" %}
-Your project will not be auto-upgraded if your environments aren't running the same **minor version**. E.g. if you are in the middle of upgrading to a new minor version, and one environment is running 8.6.x while another environment on the same project is running 8.7.x.
+Your project will not be auto-upgraded if your environments aren't running the same **minor version**. For example, when trying to upgrade a project to a new minor version where one environment is running 8.6.x and another is running 8.7.x.
 {% endhint %}
```
Comment (alina-tincas) [inline umbraco-cloud/product-upgrades/README.md:28]: to resolve the vale error
```suggestion

Your project will not be auto-upgraded if your environments aren't running the same **minor version**. For example, when trying to upgrade a project to a new minor version where one environment is running 8.6.x and another is running 8.7.x.

{% endhint %}
```

**Review by alina-tincas — COMMENTED** [review]
resolving vale error

**Comment by alina-tincas** [comment]
Awesome, merging it in 😊 🎉

File: umbraco-cloud/product-upgrades/README.md [hunk 2]
```diff
@@ -24,7 +24,9 @@ When minor upgrades are available, you will need a Development environment on yo
 * Umbraco Cloud reserves the right to roll out an emergency product fix to fix a critical issue at any time
 
 {% hint style="info" %}
-Your project will not be auto-upgraded if your environments aren't running the same **minor version**. E.g. if you are in the middle of upgrading to a new minor version, and one environment is running 8.6.x while another environment on the same project is running 8.7.x.
+
+Your project will not be auto-upgraded if your environments aren't running the same **minor version**. For example, when trying to upgrade a project to a new minor version where one environment is running 8.6.x and another is running 8.7.x.
```
Comment (alina-tincas) [inline umbraco-cloud/product-upgrades/README.md:28]:
```suggestion
Your project will not be auto-upgraded if your environments aren't running the same **minor version**. For example, when trying to upgrade a project to a new minor version where one environment is running 10.6.x and another is running 10.7.x.
```

**Review by alina-tincas — APPROVED** [review]
trying to resolve the failed check

**Pull request merged**